tables/Dysco/threadeddyscocolumn.cc

```diff
@@ -105,21 +105,20 @@ void ThreadedDyscoColumn<DataType>::loadBlock(size_t blockIndex) {
 
 template <typename DataType>
 void ThreadedDyscoColumn<DataType>::getValues(
-    casacore::uInt rowNr, casacore::Array<DataType> *dataPtr) {
+    casacore::rownr_t rowNr, casacore::Array<DataType> *dataArr) {
+  // Make sure array storage is contiguous.
+  casacore::Bool deleteIt;
+  DataType* dataPtr = dataArr->getStorage (deleteIt);
```

Comment:
Sometime in the future, it would be better to do this using a RAII structure. There are all kind of exceptions possible in the code below, which would lead to leaks. Not really directly a concern now, so no changes required, but maybe for a future MR.